src/group_theory/commutator.lean

```diff
@@ -204,3 +204,25 @@ begin
 end
 
 end subgroup
+
+variables (G)
+
+/-- The set of commutator elements `⁅g₁, g₂⁆` in `G`. -/
+def commutators : set G :=
+{g | ∃ g₁ g₂ : G, ⁅g₁, g₂⁆ = g}
```

Comment:
This PR clearly shows why we should have this definition. But I'm a bit worried that we will now have two names `commutator` and `commutators`, and the names don't really say how they differ.  Should we somehow put `set` or `subgroup` into these names?

Reply:
@jcommelin How about `commutator_set` for this so that it's not confused with the derived subgroup, and perhaps leave `commutator` alone since it's the commutator at the level of subgroups?

Reply:
I've renamed `commutators` to `commutator_set`, as suggested by Kyle.